x-pack/plugins/index_management/__jest__/client_integration/helpers/template_form.helpers.ts

```diff
@@ -135,6 +137,29 @@ export const formSetup = async (initTestBed: SetupFunc<TestSubjects>) => {
     }
   };
 
+  const clickReviewStep = async ({ name, indexPatterns }) => {
```

Comment:
This action should not depend on where we are at in the wizard IMO. Here we are assuming that we are on the logistic steps with the `nameField.input` present in the DOM.

This action should be divided in 2:

- `completeLogisticStep()`;
- `clickReviewStep()`; // Code starting L157